librz/arch/isa/mips/il/common.c

```diff
@@ -0,0 +1,28 @@
+// SPDX-FileCopyrightText: 2025 deroad <deroad@kumo.xn--q9jyb4c>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#define MIPS_DWORD_SIZE 64
+#define MIPS_WORD_SIZE  32
+#define MIPS_HALF_SIZE  16
+#define MIPS_BYTE_SIZE  8
+
+#define REG_IS_ZERO(idx) mips_reg_is_zero(insn, idx)
```

Comment:
```suggestion
#define IS_ZERO_REG(idx) mips_reg_is_zero(insn, idx)
```

Maybe better, otherwise it could refer to the value in the register.